python/lsst/jointcal/jointcal.py

```diff
@@ -920,6 +928,53 @@ def _make_one_input_data(self, visitRecord, catalog, detectorDict):
                                  filter=lsst.afw.image.FilterLabel(band=visitRecord['band'],
                                                                    physical=visitRecord['physical_filter']))
 
+    def _get_sourceTable_visit_columns(self, inColumns):
+        """
+        Get the sourceTable_visit columns from the config.
```

Comment:
I wonder if this is something we can put somewhere more general? I don't know if a tweak of this is generally applicable for code that reads the visit-level tables?